orchestra/utils/task_lifecycle.py

```diff
@@ -367,7 +369,8 @@ def get_task_details(task_id):
             'id': project.id,
             'details': project.short_description,
             'team_messages_url': project.team_messages_url,
-            'project_data': project.project_data
+            'project_data': project.project_data,
+            'status': dict(Project.STATUS_CHOICES)[project.status]
```

Comment:
sanity-check (we can chat live) --- do we want to be passing around the enum value (e.g., 0, 1, ...) or the textual value (Paused, Active, ...)